nucypher/network/nodes.py

```diff
@@ -854,38 +871,63 @@ def __init__(self,
                  certificate_filepath: str,
                  interface_signature=NOT_SIGNED.bool_value(False),
                  timestamp=NOT_SIGNED,
-                 identity_evidence=NOT_SIGNED,
+                 decentralized_identity_evidence=NOT_SIGNED,
                  substantiate_immediately=False,
                  password=None,
                  ) -> None:
 
+        #
+        # Fleet
+        #
+
         self.serving_domains = domains
-        self.certificate = certificate
-        self.certificate_filepath = certificate_filepath
-        self._interface_signature_object = interface_signature
-        self._timestamp = timestamp
-        self.last_seen = NEVER_SEEN("No Connection to Node")
         self.fleet_state_checksum = None
         self.fleet_state_updated = None
-        self._identity_evidence = constant_or_bytes(identity_evidence)
+        self.last_seen = NEVER_SEEN("No Connection to Node")
+
+        self.fleet_state_icon = UNKNOWN_FLEET_STATE
+        self.fleet_state_nickname = UNKNOWN_FLEET_STATE
+        self.fleet_state_nickname_metadata = UNKNOWN_FLEET_STATE
+
+        #
+        # Identity
+        #
+
+        self._timestamp = timestamp
+        self.certificate = certificate
+        self.certificate_filepath = certificate_filepath
+        self.__interface_signature = interface_signature
+        self.__decentralized_identity_evidence = constant_or_bytes(decentralized_identity_evidence)
+
+        # Assume unverified
+        self.verified_stamp = False
```

Comment:
Why move these settings to `__init__`?  Doesn't this pattern just put more bloat in `__init__`?